akka-docs/rst/scala/code/docs/actor/FSMDocSpec.scala

```diff
@@ -204,18 +204,18 @@ class FSMDocSpec extends MyFavoriteTestFrameWorkPlusAkkaTestKit {
       buncher ! SetTarget(testActor)
       buncher ! Queue(42)
       buncher ! Queue(43)
-      expectMsg(Batch(immutable.Seq(42, 43)))
+      expectMsg(Batch(Seq(42, 43)))
       buncher ! Queue(44)
       buncher ! Flush
       buncher ! Queue(45)
-      expectMsg(Batch(immutable.Seq(44)))
-      expectMsg(Batch(immutable.Seq(45)))
+      expectMsg(Batch(Seq(44)))
+      expectMsg(Batch(Seq(45)))
     }
 
     "not batch if uninitialized" in {
       val buncher = system.actorOf(Props(classOf[Buncher], this))
       buncher ! Queue(42)
-      expectNoMsg
+      expectNoMsg()
```

Comment:
I'm not sure about the line changes adding `()`, we like methods like `stay` and `expectNoMsg` to be kept without the parens. Would you mind keeping it as is in this line (and the `stay` line?)